BEIMA.Client/cypress/integration/DevicePage.spec.js

```diff
@@ -25,23 +25,23 @@ describe('Device Page', () => {
     cy.get('[id=deviceLatitude]').should('exist')
     cy.get('[id=deviceLongitude]').should('exist')
     cy.get('[id=locationNotes]').should('exist')
-    cy.get('[id=deviceTag]').should('exist')
-    cy.get('[id=deviceModelNumber]').should('exist')
-    cy.get('[id=deviceSerialNumber]').should('exist')
-    cy.get('[id=deviceManufacturer]').should('exist')
```

Comment:
I am confused as to why we are removing these?

Reply:
They are picked up by this statement:

cy.get('[id=fields]').within(() => {
      cy.get('input').each((val, index, collection) => {
       // test stuff here
      })
    })

Reply:
Our fields attribute should only contain the attributes specific to the device type though right? Or is fields referring to something else? In that case wouldn't you also include removing the building stuff and deviceNotes?

Reply:
Its sort of combined. Fields contains those 4 ids I removes plus the attributes specific to the device types. The building stuff isn't included as I have it grouped differently in the UI. The id fields is mostly used so that I can grab the inputs from the custom fields as they will have dynamic keys.